### DISCUSSION TITLE
Plesk: Update from 9.4.3 to 9.5.1

### DISCUSSION THREAD

**Discussion by fheuser** [directus/directus#11486] [Archive]
I built an installation following these instructions:

https://docs.directus.io/getting-started/installation/plesk/

Everything works fine, but how do I get the application updated?

**Comment by azrikahar** [comment]
Prefacing with I'm not familiar with Plesk, but I believe you should be able to add scripts listed in [the upgrading/updating project docs](https://docs.directus.io/configuration/upgrades-migrations/#upgrading-updating-a-project) in your `package.json` in the specified order.

**Reply by fheuser** [reply]
Thank you azrikahar, that was the right hint. I added this update script, which I could then start via Plesk.

<img width="692" alt="image" src="https://user-images.githubusercontent.com/6376921/152916107-13f56afd-3e0f-4597-ad62-e4973e71dfbe.png">
